src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -46,8 +48,8 @@ public partial class DSAKeyValue : System.Security.Cryptography.Xml.KeyInfoClaus
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("ios")]
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("tvos")]
         public DSAKeyValue() { }
-        public DSAKeyValue(System.Security.Cryptography.DSA key) { }
-        public System.Security.Cryptography.DSA Key { get { throw null; } set { } }
+        public DSAKeyValue(System.Security.Cryptography.DSA? key) { }
+        public System.Security.Cryptography.DSA? Key { get { throw null; } set { } }
```

Comment:
if none of the code passes in null and all usages throw consider changing key to be non-null and filing an issue on lack of throwing

Reply:
Removed nullability in both and will file an issue for missing null checks.

Reply:
https://github.com/dotnet/runtime/issues/75048